### ISSUE TITLE
Use docker networks instead of deprecated --link.

### ISSUE THREAD

**Issue by kwerey** [camptocamp/terraboard#45]
Hi folks,

This is an awesome project!! Thanks for putting it out there.

Some quick feedback on getting the docker image running locally: I had issues getting a link to a postgres container working on Docker for Mac (17.12.0-ce-mac49). It might just be that my VPN setup wasn't playing nice with it, but Docker [strongly recommends against link](https://docs.docker.com/network/links/) nowadays, and using a shared network is simple and clean to set up.

Instead, I did something along these lines and it worked a treat:

```
docker network create terranet
docker run --name db \
  -e POSTGRES_USER=gorm \
  -e POSTGRES_DB=gorm \
  -e POSTGRES_PASSWORD="<mypassword>" \
   --net terranet \
  --restart=always postgres -d
docker run -p 8080:8080 \
 -e AWS_REGION="us-east-1" \
 -e AWS_ACCESS_KEY_ID="${ACCESS_KEY_ID}" \
 -e AWS_SECRET_ACCESS_KEY="${ACCESS_KEY}" \
 -e AWS_BUCKET="<bucket>" \
 -e AWS_DYNAMODB_TABLE="<table>" \
 -e DB_PASSWORD="<mypassword>" \
 --net terranet \
 camptocamp/terraboard:latest
```

**Comment by raphink** [comment]
Sure thing. The `docker-compose` file is provided as an example, but I'd certainly welcome a PR to use v2 or v3 and avoid links!

**Comment by kwerey** [comment]
Sure - I'll see about putting something together one day next week.

**Comment by uritau** [comment]
I think tht this issue is already solved, it isn't? 🤔

**Comment by raphink** [comment]
I'll close this now since @uritau fixed it in #124
